RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# Releasing the SDK
+
+This document summarizes the steps to release the `comit-js-sdk`.
+The steps below were recorded when releasing version `0.10.0`.
+You of course have to replace `0.10.0` with your version to be released!
+
+1. Open changelog and decide new version number based on changes. 
```

Comment:
❓ Should we mention that we follow SemVer and [link](https://semver.org/) to it?